docs/ops/deployment/hadoop.md

```diff
@@ -66,6 +71,29 @@ in the shell. Note that `hadoop` is the hadoop binary and that `classpath` is an
 
 Putting the Hadoop configuration in the same class path as the Hadoop libraries makes Flink pick up that configuration.
 
+### Adding Hadoop to /lib
+
+The Flink project releases Hadoop distributions for specific versions, that relocate or exclude several dependencies
+to reduce the risk of dependency clashes.
+These can be found on the [downloads]({{ site.download_url }}).
+For these versions it is sufficient to download the corresponding `Pre-bundled Hadoop` component and placing it in
+the `/lib` directory of the Flink distribution.
+
+If the used Hadoop version is not listed on the download page (possibly due to being a Vendor-specific version),
+then it is necessary to build [flink-shaded](https://github.com/apache/flink-shaded) against this version.
+You can find the source for this project in the [Additional Components]({{ site.download_url }}#additional-components) section of the download page.
```

Comment:
The website is the primary source for the projects source code and hence shouldn't be preceded by "also". It would imply that users _alternatively_ download things from the website.